app/navigation/NotificationNavigator.scala

```diff
@@ -40,6 +40,12 @@ class NotificationNavigator @Inject()() {
 
     case AreYouTheIndividualPage => _ => routes.OffshoreLiabilitiesController.onPageLoad(NormalMode)
 
+    case OffshoreLiabilitiesPage => ua => ua.get(OffshoreLiabilitiesPage) match {
+      case Some(OffshoreLiabilities.Option1) => routes.OnshoreLiabilitiesController.onPageLoad(NormalMode)
+      case Some(OffshoreLiabilities.Option2) => routes.OnlyOnshoreLiabilitiesController.onPageLoad
```

Comment:
I think we should rename Option1 and Option2 to something more meaningful so that it's easy to understand without referring back to the object. I know that was developed under a different ticket but it would be nice to do it here